index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="main.css">
+    <title>Document</title>
+</head>
+<body>
+   
+    <main id="main">
+       <article id="billboard">
+            <nav>
+                    <ul id="nav-bar-list">
+                            <li><a href="" class=""><span>Apple</span></a></li>
+                            <li><a href=""> Store </a></li> 
+                            <li><a href=""> Mac </a></li> 
+                            <li><a href=""> iPod </a></li> 
+                            <li><a href=""> iPhone </a></li> 
+                            <li><a href=""> iPad </a></li> 
+                            <li><a href=""> iTunes </a></li> 
+                            <li><a href=""> Support </a></li> 
+                            <input type="text" id= "search-bar">
+                    </ul>
+                </nav>
+           <section id="billboard-section">
+                <a class="headline" href="https://i0.wp.com/9to5mac.com/wp-content/uploads/sites/6/2014/02/mountain.png?resize=1500%2C0&quality=82&strip=all" ><div id="headline-div"></div></a>
+                <a class="subheadline">
+                        <div id="subheadline-div">
+
+                        </div>
+                 </a>
+                <a  class="watch-film" href="">
+                       <div id="watch-film-container"></div>
+                </a>
+
+           </section>
+       </article>
+        <aside>
```

Comment:
The content in this aside tag should be in the footer, and the images should be passed as background images, just like you did for the ones above. use a div and pass as background image